### PR TITLE
fix (adev): add fallback color for gradient code

### LINKED ISSUE #52598: docs: add color fallback on safari mobile in dark theme
### Describe the problem that you experienced

I was reading the docs and it seemed like some words were missing. At first I ignored it, but after facing this problem a second time, I couldn't stand it.

Look at the screenshot provided: Defining a component -> 1.

### Enter the URL of the topic with the problem

_No response_

### Describe what you were looking for in the documentation

Some good info to learn more about Angular.

### Describe the actions that led you to experience the problem

Just reading the docs.

### Describe what you want to experience that would fix the problem

A readable documentation.

### Add a screenshot if that helps illustrate the problem

IMG_20231107_232408_761

It's fine on Android Chrome
Screenshot_2023-11-07-23-18-25-163_com android chrome

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

```true
Iphone 14 plus 
Safari browser
iOS 16
```

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by azeek21**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: documentation style fix.

## What is the current behavior?

inline code is  invisible in iOS safar browser versions < 17
Issue Number: fixes #52598 

## What is the new behavior?
inline code is visible with alike fallback color

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/shared/src/lib/styles/docs/_code.scss [hunk 1]
```diff
@@ -14,6 +14,8 @@
       position: relative;
       padding: 0 0.3rem;
       background: var(--red-to-orange-horizontal-gradient);
+      // Fallback
+      background-color: #e8b3cd;
```
Comment (bruteF04C3) [inline adev/shared/src/lib/styles/docs/_code.scss:18]: Nice work, can we use a variable for the same ?
Helps with re usability for obvious reasons. So, something along the lines of 
`--red-to-orange-horizontal-gradient-fallback`

**Review by JeanMeche — CHANGES_REQUESTED** [review]
We don't want inline colors. 
This needs a bit a refactoring and we won't review it until infra is setup.

File: adev/shared/src/lib/styles/docs/_code.scss (hunk 1 above)
Comment (azeek21) [reply adev/shared/src/lib/styles/docs/_code.scss]: Hey, sure. I  though about this before adding the color, but I was not sure if it's okay to edit the variables file.

**Comment by azeek21** [comment]
> We don't want inline colors. This needs a bit a refactoring and we won't review it until infra is setup.

You are absolutely right and I tried to refactor it. I hope it's ok this time.

File: adev/shared/src/lib/styles/docs/_code.scss [hunk 2]
```diff
@@ -14,6 +14,8 @@
       position: relative;
       padding: 0 0.3rem;
       background: var(--red-to-orange-horizontal-gradient);
+      // Fallback
+      background-color: var(--red-to-orange-horizontal-gradient-fallback);
```
Comment (twerske) [inline adev/shared/src/lib/styles/docs/_code.scss:18]: No need to add a variable here, you can use the `var(--)` directly.

**Review by twerske — CHANGES_REQUESTED** [review]
Thank you for the PR! This work is blocked until we can run the app and preview these changes.

That said, I think there are likely more fallback color locations needed.

**Comment by jessicajaniuk** [comment]
@azeek21 Looks like this now has conflicts. Can you rebase and address them? We're in a place where we should be able to preview these now and get this moving forward.

**Comment by jessicajaniuk** [comment]
Seeing no update here, I'm going to close this. Feel free to open a new PR with this change if you'd like to land it. Thanks for the contribution!

**Pull request closed without merging**